doc/sql.extensions/README.trim

```diff
@@ -18,6 +18,9 @@ Format:
 	<trim character> ::=
 		<value expression>
 
+	<multi-character trim function> ::=
+		{ BTRIM | LTRIM | RTRIM } <left paren> <trim source> [ <comma> <trim character> ] <right paren>
+
```

Comment:
Please document more explicitly difference between the new functions when using a `trim character` and specifiy that they work identically to previously function when `trim character` is not passed.

Reply:
Corrected the comments, please check how else I need to finalize the documentation